### PR TITLE
Add pulse when jobs are waiting/processing.

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

**Comment by ildyria** [comment]
@d7415 @kamil4  should I add a config to disable this ?

File: database/migrations/2024_01_24_063519_job_feedback_options.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+use App\Models\Extensions\BaseConfigMigration;
+
+return new class() extends BaseConfigMigration {
+	public const PROCESSING = 'Image Processing';
+	public const POSITIVE = 'positive';
```
Comment (kamil4) [inline database/migrations/2024_01_24_063519_job_feedback_options.php]:
```suggestion
```

Not needed?

**Review by kamil4 — APPROVED** [review]
LGTM (not tested).

File: database/migrations/2024_01_24_063519_job_feedback_options.php (hunk 1 above)
Comment (ildyria) [reply database/migrations/2024_01_24_063519_job_feedback_options.php]: yeah left over from another migration :laughing:

**Pull request merged**